ports/sonic.mania/sonicmania/Settings.ini

```diff
@@ -27,6 +27,8 @@ exclusiveFS=y
 vsync=n
 tripleBuffering=n
 pixWidth=320
+fsWidth=720
+fsWidth=240
 winWidth=720
 winHeight=240
 fsWidth=640
```

Comment:
fsWidth exists already

Reply:
Oh thanks! Not sure how I missed this below 🤦‍♀️